brave/brave-core
# Adds delays for suggestion api

Resolves https://github.com/brave/brave-browser/issues/7709
Resolves https://github.com/brave/brave-browser/issues/6940

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards (with short contribution)
- claim promotion
- add couple of monthly contributions
- add couple of verified publishers to ac table
- wait for contribution to be triggered
- make sure that everything goes through and that reports and balance is reflected correctly

- enable rewards
- claim promotion
- do one time tips
- make sure that everything is ok (summary and balance)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
